src/main/java/seedu/address/ui/StatisticsDisplay.java

```diff
@@ -12,69 +14,129 @@
-    @FXML private Label barChartTitle;
-    @FXML private BarChart<String, Integer> barChart;
+    @FXML private BarChart<String, Integer> barChartPomDurationData;
+    @FXML private BarChart<String, Integer> barChartTasksDoneData;
 
     @FXML private Label progressDaily;
     @FXML private ImageView progressBarDaily;
     @FXML private Label medals;
 
     public StatisticsDisplay() {
         super(FXML);
-        this.barChartTitleText = DEFAULT_BAR_CHART_TITLE;
-        this.data = null; // TODO *******************
-        this.progressBarDailyFilepath = DEFAULT_PROGRESS_BAR_DAILY__PLACEHOLDER;
+        this.progressBarDailyFilepathString = DEFAULT_PROGRESS_BAR_DAILY_PLACEHOLDER;
+
         this.progressDailyText = DEFAULT_PROGRESS_DAILY;
         this.medalsText = DEFAULT_MEDALS;
 
         progressDaily.setText(progressDailyText);
-        Image progressBarDailyImage = new Image(String.valueOf(progressBarDailyFilepath));
+        Image progressBarDailyImage = new Image(DEFAULT_PROGRESS_BAR_DAILY_PLACEHOLDER);
         progressBarDaily.setImage(progressBarDailyImage);
         medals.setText(medalsText);
 
-        barChartTitle.setText(barChartTitleText);
+        CategoryAxis xAxis1 = new CategoryAxis();
+        xAxis1.setLabel(X_AXIS);
+        NumberAxis yAxis1 = new NumberAxis();
 
-        CategoryAxis xAxis = new CategoryAxis();
-        xAxis.setLabel(X_AXIS);
-        NumberAxis yAxis = new NumberAxis();
-        yAxis.setLabel(POMODORO_RUNTIME_Y_AXIS);
+        CategoryAxis xAxis2 = new CategoryAxis();
+        xAxis2.setLabel(X_AXIS);
+        NumberAxis yAxis2 = new NumberAxis();
+    }
 
-        XYChart.Series<String, Integer> dataSeries1 = new XYChart.Series<>();
-        dataSeries1.setName("You");
+    public void updateGraphs(CustomQueue customQueue) {
+        DayData latestDayData = customQueue.get(CONSTANT_SIZE - 1);
+        int currProgress = latestDayData.getPomDurationData().value;
+        progressDaily.setText(currProgress + PROGRESS_UNITS);
 
-        // TODO remove ********************
-        dataSeries1.getData().add(new XYChart.Data<>("Day 1", 20));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 2", 40));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 3", 60));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 4", 40));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 5", 20));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 6", 40));
-        dataSeries1.getData().add(new XYChart.Data<>("Day 7", 80));
+        int expBarPerc = currProgress / 10;
+        if (expBarPerc >= 10) {
+            expBarPerc = 10;
+        }
 
-        barChart.getData().add(dataSeries1);
-    }
+        switch (expBarPerc) {
+            case 0:
+                this.progressBarDailyFilepathString = "/images/pet/ProgressBar0%.png";
+                break;
+            case 1:
+                this.progressBarDailyFilepathString = "/images/pet/ProgressBar10%.png";
```

Comment:
maybe a String.format()?
String.format("/images/pet/ProgressBar%d0%.png", expBarPerc); 
Except for the 0 one, unless you rename the filename